src/Aspire.Dashboard/Components/Pages/ConsoleLogs.razor.cs

```diff
@@ -150,6 +154,10 @@ protected override async Task OnInitializedAsync()
             PageViewModel.Status = Loc[nameof(Dashboard.Resources.ConsoleLogs.ConsoleLogsLogsNotYetAvailable)];
         }
 
+        await this.InitializeComponentTelemetryAsync();
+
+        return;
+
```

Comment:
This doesn't do anything. Below it are local methods.
```suggestion
```

Reply:
I prefer to keep the return before local methods for readability. But deleted